### ISSUE TITLE
Integration with wcag-validator?

### ISSUE THREAD

**Issue by jatinsi** [bbc/color-contrast-checker#1]
Can this module be integrated with wcag-validator? if yes please guide me how. I want to use it for accessibility testing.
Wcag-validator does not have the contrast module in it so this seems to be the best fit but I have no idea how to build a js project so need guidance.

**Comment by Qambar** [comment]
hi jatinsi, both are two different projects. So you would use them as separate libraries in your project.
